modules/core/src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Tracker.scala

```diff
@@ -386,41 +365,11 @@ class Tracker(emitters: Seq[TEmitter], namespace: String, appId: String, encodeB
    * Set the Subject for the tracker
    * The subject's configuration will be attached to every event
    *
-   * @param subject user which the Tracker will track
+   * @param newSubject user which the Tracker will track
    * @return The tracker instance
    */
-  def setSubject(subject: Subject): Tracker = {
-    this.subject = subject
-    this
-  }
-
-  /**
-   * Adds EC2 context to each sent event
-   * Blocks event queue until either context resolved or timed out
-   */
-  def enableEc2Context(): Unit = {
```

Comment:
These are moved to the `metadata` module

Reply:
Big 👍